qt_ui/windows/basemenu/QBaseMenu2.py

```diff
@@ -241,9 +241,47 @@ def closeEvent(self, close_event: QCloseEvent):
         GameUpdateSignal.get_instance().updateGame(self.game_model.game)
 
     def get_base_image(self):
+        try:
+            game = self.game_model.game
+        except:
+            game = None
+
         if self.cp.cptype == ControlPointType.AIRCRAFT_CARRIER_GROUP:
+            for ground_object in self.cp.ground_objects:
+                for group in ground_object.groups:
+                    for unit in group.units:
+                        if unit.type == "Forrestal":
+                            return "./resources/ui/carrier_cv59.png"
+                        elif (
+                            unit.type == "Stennis"
+                            and game is not None
+                            and game.settings.supercarrier
+                            or unit.type == "CVN_71"
+                            or unit.type == "CVN_72"
+                            or unit.type == "CVN_73"
+                            or unit.type == "CVN_75"
```

Comment:
This is getting messy. This sort of thing should be factored out into unit data yaml files. We don't currently have unit data yaml for naval units, but those should be added. For now just a mapping from unit type to the banner is fine.

Example `resources/units/ships/CVN_75.yaml`:
```yaml
---
control_point_banner: resources/ui/carrier_cvn75.png
```

Reply:
Alternatively, just use `resources/ui/units/ships/{unit.type}.png` and add an image per unit type (they can all be copies if you don't want to track down unique images for each ship right now).

Reply:
I added images for the rest of the carriers and moved them to resources/ui/units/ships/ as requested.